### PR TITLE
Update frontend email aesthetics to include name as well as colour, button and padding changes

### PR DESCRIPTION
**Pull request by ABarroso647**
image
outlook, dark mode (outlook dark mode will auto update colours)
sample iphone
image
sample samsung
image
sample gmail/lightmode 
image

"me" is the name here

### CODE REVIEW

File: backend/html/emailTemplate.html [hunk 1]
```diff
@@ -4,47 +4,36 @@
...
+    <tr>
+        <td>
+<![endif]-->
+<table class="desktop-only" style="width: 100%; max-width: 600px; margin: 0 auto; background-color: #ffffff; padding: 20px; border-radius: 8px; border: 2px solid #000000; border-collapse: collapse;">
+    <tr>
+        <td style="padding-left: 30px; margin-left: 5px;">
+            <h1 style="font-weight: bold; color: #000000;">Key registration request for Lynx Locks</h1>
+            <p style="color: #000000; margin-bottom: 20px;">Hi, {{.Name}}, please use the below link to add a key to your Lynx Locks account.</p>
+            <p style="color: #000000; margin-bottom: 20px;">Be sure to use it on a device capable of registering passkeys or a YubiKey!</p>
+            <a href="{{.Link}}" style="display: inline-block; background-color: rgb(242, 155, 4); color: #000000; text-decoration: none; padding: 15px 25px; border-radius: 8px; font-weight: bold; font-size: 16px;">Add Your Key</a>
```
Comment (AdrianJendo) [inline backend/html/emailTemplate.html:28]: Does <button> not work on this?

Comment (AdrianJendo) [reply backend/html/emailTemplate.html:28]: Why is the orange different on light / dark? Also why does the first one have no padding?

Comment (AdrianJendo) [reply backend/html/emailTemplate.html:28]: Also why did you change all the styling to be inline?

Comment (ABarroso647) [reply backend/html/emailTemplate.html:28]: idk what ur first comment is, the 2nd is its literally just outlook being weird, it shouldn't be different,
the styling is inline as sometimes outlook has a hard time rendering complex styling hence this was an attempt, and then further changes were done from there so it was left

Comment (ABarroso647) [reply backend/html/emailTemplate.html:28]: the issue with the orange is probably (definitely) cuz outlook darkmode isn't real darkmode they just take your colours and invert them

Comment (ABarroso647) [reply backend/html/emailTemplate.html:28]: also idk why one u said has no padding, i think all my images do?

Comment (AdrianJendo) [reply backend/html/emailTemplate.html:28]: Screenshot 2024-03-13 at 10 58 45 PM

Comment (ABarroso647) [reply backend/html/emailTemplate.html:28]: ah i see the button, yea another outlook special if you use new outlook everything is fine
image
should've added that in my description mb

Comment (AdrianJendo) [reply backend/html/emailTemplate.html:28]: Why do you have all the styling inline?

Comment (ABarroso647) [reply backend/html/emailTemplate.html:28]: It was supposed to be easier for emails to interpret, (not sure how much affect it actually ended up happening) but that is why it was transitioned and then why it was left that way as per wat i said earlier

**Review by AdrianJendo — APPROVED** [review]

**Review by cameronkinsella — APPROVED** [review]

**Pull request merged**